packages/tailwindcss/src/compile.ts

```diff
@@ -310,30 +314,40 @@ function applyImportant(ast: AstNode[]): void {
 
 function getPropertySort(nodes: AstNode[]) {
   // Determine sort order based on properties used
-  let propertySort = new Set<number>()
+  let order = new Set<number>()
+  let count = 0
   let q: AstNode[] = nodes.slice()
 
+  let seenTwSort = false
+
   while (q.length > 0) {
     // SAFETY: At this point it is safe to use TypeScript's non-null assertion
     // operator because we guarded against `q.length > 0` above.
     let node = q.shift()!
     if (node.kind === 'declaration') {
-      if (node.property === '--tw-sort') {
-        let idx = GLOBAL_PROPERTY_ORDER.indexOf(node.value ?? '')
-        if (idx !== -1) {
-          propertySort.add(idx)
-          break
+      count++
+
+      if (!seenTwSort) {
```

Comment:
We were using a `break` before once we noticed a `--tw-sort`, but now we want to make sure we count _all_ properties. So once we notice a `--tw-sort` we don't want to collect new properties in the `order` set anymore, but we still want to keep walking to count other properties.

Reply:
Could we invert this condition so we reduce the level of nesting here? 

```suggestion
      if (teenTwSort) continue;
```